File: APIs/py/universitat/models/professor.py
from . import db

class Professor(db.Model):
    DNI = db.Column(db.String(10), primary_key=True)
    nom = db.Column(db.String(50), nullable=False)
    email = db.Column(db.String(120), nullable=False)

    def __repr__(self):
        return f'<Professor {self.nom}>'

File: APIs/.js/Node.js/universitat/models/Professor.js
const mongoose = require('mongoose');

const professorSchema = new mongoose.Schema({
  dni: { type: String, required: true, unique: true },
  nom: { type: String, required: true },
  cognoms: { type: String, required: true },
  departament: { type: String, required: true },
});

module.exports = mongoose.model('Professor', professorSchema);
